cinema/migrations/0005_alter_movie_actors_alter_movie_genres.py

```diff
@@ -0,0 +1,23 @@
+# Generated by Django 4.0.4 on 2024-11-19 10:35
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('cinema', '0004_alter_genre_name'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='movie',
+            name='actors',
+            field=models.ManyToManyField(related_name='movies', to='cinema.actor'),
+        ),
+        migrations.AlterField(
+            model_name='movie',
+            name='genres',
+            field=models.ManyToManyField(related_name='movies', to='cinema.genre'),
```

Comment:
The `related_name` 'movies' is used for both the `actors` and `genres` fields. This can cause conflicts when accessing related objects. Consider using unique `related_name` values for each field to avoid these conflicts.